client/src/api/locations.js

```diff
@@ -0,0 +1,11 @@
+export async function getLocation() {
+  const response = await fetch("http://localhost:2601/locations");
```

Comment:
This won't work in production anymore, please use relative paths
```suggestion
  const response = await fetch("/locations");
```